src/commands/gate/renderer.ts

```diff
@@ -18,7 +18,11 @@ export const renderEvaluationResponse = (evaluationResponse: EvaluationResponse)
   }
 
   let fullStr = ''
-  fullStr += chalk.green('Successfully evaluated all matching rules.\n')
+  fullStr += chalk.green(
```

Comment:
Rather than doing this, I would do for dry-run the same thing we are doing for empty, i.e. catching it in a if statement above and render something specific in that case. That allows us to modify thing in "normal" states without thinking on whether the dry-run case supports is or not. So something like:

```
if (evaluationResponse.status.toLowerCase() === 'empty') {
    return renderDryRunEvaluation(evaluationResponse)
  }
```

With the `renderDryRunEvaluation` method printing some specific dry-run information

Reply:
I have created a separate rendering function for dry run responses. There is some repeated code, but it is now easier to make changes in rendering differentiating  by dry run / no dry run.